src/lib/dnssd/Resolver_ImplMinimalMdns.cpp

```diff
@@ -418,6 +418,10 @@ void MinMdnsResolver::AdvancePendingResolverStates()
                 mActiveResolves.CompleteCommissionable(nodeData);
                 nodeData.nodeType = DiscoveryType::kCommissionableNode;
                 break;
+            case IncrementalResolver::ServiceNameType::kOperational:
```

Comment:
Is there something tracking adding this to the "platform" dns-sd implementation?

Reply:
Since the platform implementation of operational discovery in #28301 is blocked on some discussion, it is not considered here.